js/okcoin.js

```diff
@@ -2764,32 +2761,32 @@ module.exports = class okcoin extends Exchange {
         // or a negative number (transaction fee deduction)
         // therefore we need to invert the fee
         // more about it https://github.com/ccxt/ccxt/issues/5909
-        if ((feeCostFirst !== undefined) && (feeCostFirst !== 0)) {
-            if ((feeCostSecond !== undefined) && (feeCostSecond !== 0)) {
+        if ((feeCostFirstString !== undefined) && (feeCostFirstString !== '0')) {
```

Comment:
Let's use `Precise.stringEq` for all comparisons, cause '0' could be '0.0' or '0.00000000', and all those strings are the different, but the number is the same.

Reply:
```JavaScript
if ((feeCostFirstString !== undefined) && Precise.stringEq (feeCostFirstString, '0')) {
```

Please, also change all occurrences below.